.gitignore

```diff
@@ -51,14 +51,6 @@ lib/CMakeFiles/
 lib/cmake_install.cmake
 
 Documentation/Examples/*.generated
-Documentation/Books/Users/book.json
```

Comment:
From what source file would it be generated?! It contains a title and description... I'm actually baffled how it made it onto the ignore list.